### PR TITLE
Janitoring

### PR DESCRIPTION
**Pull request by vargac**
- Replace #defines
- Set tab size to 4
- Use C++ comments
- Shorten line size to 80 columns

Signed-off-by: vargac simon.varga123@gmail.com

### CODE REVIEW

**Comment by vargac** [comment]
@mrshu ping

File: PixyViSy.cpp [hunk 1]
```diff
@@ -1,91 +1,116 @@
...
-              max[1] = max[0];
-              max[0] = i;
-            } else {
-              max[2] = max[1];
-              max[0] = i;
+    if (all_blocks_count) {
+        // finding three max blocks
+        for (uint8_t i = 0; i < all_blocks_count; i++) {
+            if (pixy.blocks[i].signature == signature
+            && block_size(pixy.blocks[i]) >= min_block_area) {
```
Comment (mrshu) [inline PixyViSy.cpp:38]: @vargac To be honest, I am not sure if this is an actual improvement, in terms of readability.

File: PixyViSy.cpp [hunk 2]
```diff
@@ -1,91 +1,116 @@
 #include <PixyViSy.h>
 
-PixyViSy::PixyViSy(uint8_t sig, uint16_t calibration_dis_pix, uint8_t _goal_height)
+const uint16_t min_block_area = 100;
```
Comment (mrshu) [inline PixyViSy.cpp:3]: I am also not sure if this is better than doing it via `#define` but I trust you have a good reason for that =).

**Review by mrshu — COMMENTED** [review]
Generally this looks good. I do not see any showstoppers so feel free to merge this if you do not think any changes are necessary.

**Pull request merged**